exercises/simple-cipher/example.rs

```diff
@@ -34,8 +34,8 @@ fn shift(key: &str, s: &str, dir: i8) -> Option<String> {
     }
     for c in s.chars() {
         let shift = key_arr[i % key_arr.len()] as i8 - 'a' as i8;
-        let n = ((c as i8 - 'a' as i8 + dir * shift) % 26 + 26) % 26;
-        o.push(char::from('a' as u8 + n as u8));
+        let n = ((c as i8 - b'a' as i8 + dir * shift) % 26 + 26) % 26;
```

Comment:
I don't think Clippy had anything to say about this one, but let's do it anyway for consistency

Reply:
Exactly.